components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/action/management/v1/ActionsApiService.java

```diff
@@ -46,6 +46,8 @@ public interface ActionsApiService {
 
       public Response getActionTypes();
 
+      public Response getActionsByActionId(String actionType, String actionId);
```

Comment:
```suggestion
      public Response getActionByActionId(String actionType, String actionId);
```
Since we are retrieving one action let's use the singular term

Reply:
Addressed with commit 98b19c3b4ec9733fe59c408728bee3f58a266728